lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -418,7 +422,7 @@ int amf_idle_mode_procedure(amf_context_t* amf_ctx) {
  **                                                                        **
  **                                                                        **
  ***************************************************************************/
-void amf_free_ue_context(ue_m5gmm_context_s* ue_context_p) {
+void amf_free_ue_context(std::shared_ptr<ue_m5gmm_context_t> ue_context_p) {
```

Comment:
Update documentation of this function accordingly

Reply:
addressed review comment